.env

```diff
@@ -38,3 +38,4 @@ FLAG_WALLET_CONNECT_ENABLED=1
 API_LOGGING_ENABLED=1
 SENTRY_ENABLED=0
 SENTRY_ENVIRONMENT=ci-main
+USE_DOWNLOAD_AS_LOG_DIR=0
```

Comment:
Quick one: the directory `Downloads` is named in the plural, so the var could be named `USE_DOWNLOADS_AS_LOG_DIR`.

Another naming option is: `USE_PUBLIC_LOG_DIR` or `WRITE_LOGS_TO_PUBLIC_DIR`, because in essence it doesn't matter whether it's the `Downloads` directory as much, but the fact that it's publicly accessible to any other app.